pkgs/applications/networking/browsers/yandex-browser/default.nix

```diff
@@ -136,12 +136,5 @@ stdenv.mkDerivation rec {
     sourceProvenance = with sourceTypes; [ binaryNativeCode ];
     maintainers = with maintainers; [ dan4ik605743 ionutnechita ];
     platforms = [ "x86_64-linux" ];
-
-    knownVulnerabilities = [
-      ''
-      Trusts a Russian government issued CA certificate for some websites.
-      See https://habr.com/en/company/yandex/blog/655185/ (Russian) for details.
-      ''
-    ];
```

Comment:
Has this issue been resolved? I see no mention of it in the change set.

Reply:
Indeed, upstream doesn't seem to have intention to change it either.

Reply:
I do not think it has, I briefly checked and I find various mention that Yandex.Browser still support it out of the box ([example 1](https://mkb.ru/en/cert/win)).

I would favorable to re-introduce back the warning at very least because they do not conform with the transparency requirements set by the public root certificate programs.